### DISCUSSION TITLE
I often get error on webgl.

### DISCUSSION THREAD

**Discussion by anis-byanjankar** [thumbmarkjs/thumbmarkjs#68] [Q&A]
Webgl does not return hash persistently. Sometimes, it does return hash while at times it returns unsupported on a same browser in a span of like a minute. What could be the problem ? 

```
   {
      'webgl': 'unsupported'
    }
```

**Comment by ilkkapeltola** [comment]
Hmm... hard to say. Which browser are you using?
How reliably can you reproduce this?
